src/server/string_family_test.cc

```diff
@@ -399,4 +399,58 @@ TEST_F(StringFamilyTest, GetDel) {
   ASSERT_THAT(resp, ArgType(RespExpr::NIL));
 }
 
+TEST_F(StringFamilyTest, GetEx) {
+  using std::chrono::duration_cast;
+  using std::chrono::milliseconds;
+  using std::chrono::seconds;
+  using std::chrono::system_clock;
+
+  auto resp = Run({"set", "foo", "bar"});
+  ASSERT_THAT(resp, "OK");
+
+  // resp = Run({"getex", "foo", "EX", "0"});
+  // ASSERT_THAT(resp, ErrArg("invalid expire time"));
+
+  // resp = Run({"getex", "foo", "EX", "-1"});
+  // ASSERT_THAT(resp, ErrArg("invalid expire time"));
+
+  EXPECT_EQ(Run({"getex", "foo"}), "bar");
```

Comment:
have you checked that the behavior modeled in the tests reflect the golden standard (aka Redis 6)?

did you actually tested these sequences with Redis 6?